### PR TITLE
Rum 20850 add support for ie11 for web js sdk

### PR DESCRIPTION
**Pull request by robertdumitrescu**

### CODE REVIEW

File: config/webpack.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    exclude: /node_modules/,
+    use: {
+      loader: 'babel-loader',
+      options: {
+        presets: [
+          [
+            '@babel/preset-env',
+            {
+              targets: {
+                browsers: ['last 2 versions', 'ie >= 11']
```
Comment (yghern) [inline config/webpack.js:83]: according to our own README we support older browsers than "last 2 releases", like FF v60+ - should we update readme as well?

File: examples/Vanilla/CustomCollector/index.html [hunk 2]
```diff
@@ -13,7 +13,18 @@ <h1>Custom Collector</h1>
     <button autofocus onclick="window.location.href='../Error/index.html'">
       Next
     </button>
-    <script src="https://sdk.real-user-data.eggplant.cloud/latest/rci.min.js></script>
```
Comment (yghern) [inline examples/Vanilla/CustomCollector/index.html:16]: why did we removed publicly available URL in favor of the local path?

File: dist/rci.min.js [hunk 3]
```diff
@@ -1,20 +1,3846 @@
...
-/*!
- * UAParser.js v0.7.19
- * Lightweight JavaScript-based User-Agent string parser
- * https://github.com/faisalman/ua-parser-js
+/*! Scope: core - @eggplantio/real-user-data-sdk-js - 1.0.29 compiled at 2020-02-27T16:37:59.271Z */
+window["rciSdk"] =
```
Comment (MalcolmSmithUK) [inline dist/rci.min.js]: @robertdumitrescu MAINAINABILTY Are we not planning to remove the dist?

File: examples/Vanilla/OnLoad/index.html [hunk 4]
```diff
@@ -13,7 +13,18 @@ <h1>OnLoad</h1>
     <button autofocus onclick="window.location.href='../Conversion/index.html'">
       Next
     </button>
-    <script src="https://sdk.real-user-data.eggplant.cloud/latest/rci.min.js"></script>
-    <script type="module" src="./main.js"></script>
+    <script src="./../../../dist/rci-legacy.min.js"></script>
```
Comment (MalcolmSmithUK) [inline examples/Vanilla/OnLoad/index.html:16]: @robertdumitrescu INFO - All of the examples now use the legacy version - is this intended?

**Review by MalcolmSmithUK — CHANGES_REQUESTED** [review]

**Review by yghern — CHANGES_REQUESTED** [review]

File: dist/rci.min.js (hunk 3 above)
Comment (robertdumitrescu) [reply dist/rci.min.js]: Yes we are. I will remove it

File: examples/Vanilla/OnLoad/index.html (hunk 4 above)
Comment (robertdumitrescu) [reply examples/Vanilla/OnLoad/index.html:16]: Well yes, since those will work everywhere.

File: config/webpack.js (hunk 1 above)
Comment (robertdumitrescu) [reply config/webpack.js:83]: We need to update the readme. Maybe we can say something last 2 versions on all the columns to future proof it?

File: examples/Vanilla/CustomCollector/index.html (hunk 2 above)
Comment (robertdumitrescu) [reply examples/Vanilla/CustomCollector/index.html:16]: I will re add that as a commented line. By default it should stay targeting the dist one but we should have the option to point it to cloud one.

Comment (robertdumitrescu) [reply examples/Vanilla/CustomCollector/index.html:16]: Was already there commented out. No action taken

File: config/webpack.js (hunk 1 above)
Comment (robertdumitrescu) [reply config/webpack.js:83]: Made the changes

File: dist/rci.min.js (hunk 3 above)
Comment (robertdumitrescu) [reply dist/rci.min.js]: Removed

**Review by MalcolmSmithUK — APPROVED** [review]

**Review by yghern — APPROVED** [review]

**Pull request merged**